platform/checkout/cisco-8000.ini

```diff
@@ -1,3 +1,3 @@
 [module]
 repo=git@github.com:Cisco-8000-sonic/platform-cisco-8000.git
-ref=202405.0.15
```

Comment:
Hi @anamehra, currently only 202405 branch build break, all other branches are healthy. why this PR send to master branch?

I found in other branch, there is a minor version change which fix the build issue, can 202405 branch also have a minor version? for example 202405.0.16

Reply:
I fixed master last week, could you please check if that fine now?
I fixed 202405 today. Will fix 202311 as well today.